src/net.cpp

```diff
@@ -2606,6 +2601,19 @@ void CConnman::GetNodeStats(std::vector<CNodeStats>& vstats) const
     for (CNode* pnode : m_nodes) {
         vstats.emplace_back();
         pnode->CopyStats(vstats.back());
+
+        if (auto min_ping_time = m_evictionman.GetMinPingTime(pnode->GetId())) {
+            vstats.back().m_min_ping_time = *min_ping_time;
+        }
+
+        if (auto last_block_time = m_evictionman.GetLastBlockTime(pnode->GetId())) {
+            vstats.back().m_last_block_time = *last_block_time;
+        }
+
+        if (auto last_tx_time = m_evictionman.GetLastTxTime(pnode->GetId())) {
+            vstats.back().m_last_tx_time = *last_tx_time;
```

Comment:
`m_evictionman.CopyEvictionStats(pnode->GetId(), vstats.back());` ?

Reply:
Yea something like that would make sense. Also similar to John's [comment](https://github.com/bitcoin/bitcoin/pull/25572#discussion_r919919355).